EdvClient.js

```diff
@@ -77,7 +80,7 @@ export class EdvClient {
    * @returns {undefined}
 
    */
-  ensureIndex({attribute, unique = false}) {
+  ensureIndex({attribute, unique = false} = {}) {
```

Comment:
This change allows the method to be called with no parameters. The `attribute` parameter will be undefined and `unique` will be set to false. This will be passed to the `indexHelper.ensureIndex` function. Will calling that function with an undefined attribute result in an error? Did we want the error to be thrown from that function instead? Was this intentional?

Reply:
`indexHelper.ensureIndex` handles the error. I do see that it should have a default `{}` for the options too, so I'll add that. What we want is a good error indicating that `attribute` is required vs. an abstruse destructuring error when someone forgets to set the params.